docs/tutorials/subscription-platform.md

```diff
@@ -109,14 +109,115 @@ Some of the metadata properties listed above [have defaults][product-details-def
 
 [product-details-defaults]: https://github.com/mozilla/fxa/blob/main/packages/fxa-shared/subscriptions/metadata.ts#L14
 
-###### Legal Document Download URL Metadata
+##### Product Configuration Documents
 
-For the legal document download URL metadata,
-`product:termsOfServiceDownloadURL` and `product:privacyNoticeDownloadURL`,
-they can be in the form of an incomplete URL, as they will be handled by a
-redirect endpoint that tries to best match the user's locale to a localized
-version of the document.  For example, if the value of
-`product:termsOfServiceDownloadURL` is
+:::info
+This section is for an upcoming feature that is not yet in production.  Please
+continue to use [Stripe product metadata](#stripe-product-metadata) to
+configure your subscription products.
+:::
+
+While the [Stripe product and plan metadata](https://stripe.com/docs/api/metadata) has enable the Subscription Platform to quickly develop many features, it has multiple shortcomings.
+
+* There are limits on the number of entries and the size of the keys and
+  values.  The limit for the number of entries is rather low.
+* The strings only key-value pair format necessitates potentially confusing key
+  formats and key parsing in code.
+* A reliance on Stripe metadata for subscription product configurations means
+  the Subscription Platform need to define plans in Stripe for subscription
+  plans that are not Stripe based (e.g. App Store subscriptions).
+
+The overcome these limits, the team is moving to [JSON](https://www.json.org)
+document based product configurations.  For readability, the configuration is
+described below in multiple sections.  To see the overall format of the
+configuration documents, take a look at the following sample documents:
+* [Sample Product Configuration Document](../assets/product-configs/sample-product-config-doc.json)
+* [Sample Plan Configuration Document](../assets/product-configs/sample-plan-config-doc.json)
+
+:::tip
+A value in a plan configuration can be used to override the product configuration value.  Another way to think of this is that the values in the product configuration are the defaults for the product's plans.
+:::
+
+:::info
+The Subscription Platform team is currently developing the process to update these configuration documents.  Documentation for that is forthcoming.
+:::
+
+###### Top Level Configuration Document Values
+
+The configuration document is a JSON object.  The top level keys for that object are below.
+
+| Key                | Description |
+| ------------------ | ----------- |
+| id                 | An identifier for the system storing the document.  It's not intended for manual edits.|
+| productConfigId    | _Plan configuration only_.  The identifier of the product configuration document from which the plan configuration inherits its default values. |
```

Comment:
Some parameters are listed as `Optional` others as `Required` and some have none. The one's with none might be a little confusing whether or not its `Optional` or `Required`.

Reply:
I'll remove the `Required` and mentioned that they are required unless otherwise noted.